test/journeys/specs/oneOnOne/messaging.js

```diff
@@ -114,13 +122,94 @@ describe(`Widget Space: One on One`, () => {
 
     it(`sends and deletes message`);
 
-    it(`sends message with pdf attachment`);
+    describe(`File Transfer Tests`, () => {
+      /* eslint no-sync: "off" */
+      const uploadDir = path.join(__dirname, `assets`);
+      const downloadDir = path.join(os.homedir(), `Downloads`);
+      const inputFileElem = `.ciscospark-file-input`;
+      const downloadFileButton = `(//div[@title="Download this file"]/parent::button)[last()]`;
+      const shareButtonElem = `button[aria-label="Share"]`;
+      function sendFileTest(fileName) {
+        clearEventLog(browserRemote);
+        const filePath = path.join(uploadDir, fileName);
+        const fileSize = fs.statSync(filePath).size;
+        const downloadedFile = path.join(downloadDir, fileName);
+        const fileTitle = `//div[text()="${fileName}"]`;
+        if (fs.existsSync(downloadedFile)) {
+          fs.unlinkSync(downloadedFile);
+        }
+        browserLocal.chooseFile(inputFileElem, filePath);
+        browserLocal.click(shareButtonElem);
+        browserRemote.waitForExist(fileTitle, 30000);
+        browserRemote.moveToObject(downloadFileButton);
```

Comment:
Now that I think about it, let's `moveToObject` the parent container that causes the button to appear. This will be what the user will be doing as well. The user wouldn't know to move to the hidden button and I'm worried that we are taking advantage of a webdriver bug.